README.md

```diff
@@ -148,6 +142,70 @@ Use Qt Creator as before
 </pre>
 
 
+Usage
+=====
+
+First use - optimise
+--------------------
+
+It is strongly recommended to run the `volk_profile` utility before
+running gqrx. This will detect and enable processor-specific optimisations and
+will in many cases give a significant performance boost.
+
+The first time you start gqrx it will open a device configuration dialog.
+Supported devices that are connected to the computer are discovered
+automatically and you can select any of them in the drop-down list.
+
+If you don't see your device listed in the drop-down list it could be because:
+- The driver has not been included in a binary distribution
+- The udev rule has not been properly configured
+- Linux kernel driver is blocking access to the device
+
```

Comment:
One cause could also be that the hardware is not connected..

Reply:
Ok, I'll add that to the list too.